### DISCUSSION TITLE
What is a Centroid and how do I find it for Rad Zero 2 ?

### DISCUSSION THREAD

**Discussion by geovillage** [frntc/RAD#38] [General]
Hello.  I got angry after purchasing a Rad pi hat (no pi) for more than $40. (I know, not so smart.) So when I heard that there was a RAD for Pi Zero 2w and I have several around, I decided to go the PCBWay and have them assemble it too. (I am a senior now and hands are not steady enough for surface mounts.) Well, to make a long story even longer, they took my cash but told me they can't cancel the order, nor can they complete it, because there is no "Centroid" file.  Anyone know how I can get, make, buy such a thing?  I have already promised a few people I would get them a RAD and now I am afraid my noobish nature has me embarrassed.

**Comment by futureretrofusion** [comment]
You can use a pizero2w on any rad PCB . The PCB for the pi3 will work with
the pizero2


The file they want is probably for placement of the components on the PCB.

**Reply by geovillage** [reply]
I did figure out that I could use the pi zero 2 on the other board, but the boards I was seeing sold for both pis were for upwards of 40 bucks for just the hat. If PCBWay can complete my order of 20, assembled will be for around 8 dollars each, including shipping.  I only need a few, but the price per 20 was the sweet spot.  I intended to sell the extras at close to costs.  (probably around $10 ea.)  

Small update:  I asked PCBWay if they could provide the Centroid file.  They said they would reach out to the author.  Since this is not the only shared project missing a centroid file, I can't figure out why PCBWay doesn't warn you before you purchase.

**Reply by frntc** [reply]
They didn't reach out yet, but unfortunately I also don't have a PCBWay-compatible centroid file. Theoretically I can create one (if I find the time), but before making this available I would want to produce a test-batch myself. However, the costs for doing that for every single project are non-negligible :-/ I don't know why PCBWay offers these PCBs prebuilt without having the centroid files. I think in a former case they had an engineer create them themselves, not sure if this works this time. I think you should be able to get your payment back if they are not able to produce what they let you order. Once they reach out to me, I'm happy to tell them. Maybe this helps a bit: restore-store.de also has RADs on stock.

**Reply by geovillage** [reply]
Thanks to everyone for replying.  PCBWay reached out to me late yesterday and everything is solved (this time.)
**Here is what they said:**
_Hi, after checking, we will follow the value marking and part number marking in silkscreen to solder it by hand. For bulk order, it needs extra cost to generate pick&place file by soldering pads. But for this sample order, we will solder it by hand_
This is good to know for the future.  I guess sellers who want to make large runs will get the centroid file made for them.  For someone like me, I will learn how to generate a centroid file before ordering again.  It may still be worth it to learn as reasonable prices are hard to find on this side of the Atlantic.  

Let me end this thread by thanking you so much for creating this and other projects.   It keeps these old computers alive.

**Reply by frntc** [reply]
Guessing from previous observations: they might just record what they do and then have the centroid file ready next time :-)

(good for everyone, but not for "sellers with commercial intend" as the license doesn't permit that... your order and forwarding units close to costs if of course fine! let us know how it goes!)
